Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.4
+
+# Run as non-root, eventually
+RUN groupadd -r slamon && useradd -r -d /workspace -m -g slamon slamon 
+
+# Install dependencies
+RUN pip install requests python-dateutil responses
+
+# Install SLAMon Python Agent
+ADD . /workspace
+WORKDIR /workspace
+RUN pip install .
+RUN chown -R slamon:slamon /workspace
```

Comment:
Should the sources be removed from the image after pip install? Also not sure if a home directory for the user is required, as the agent is currently stateless and not storing anything.